### DISCUSSION TITLE
This might be a silly question, but does this package support Gemini?

### DISCUSSION THREAD

**Discussion by tunyawat** [RageAgainstThePixel/com.openai.unity#243] [Q&A] [answered]
I came across this link and am wondering whether it could be used with your package?

Call Gemini by using the OpenAI library
https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/call-gemini-using-openai-library

**Comment by StephenHodgson** [comment, marked as answer]
No it does not.

Google does have a good library generated for .net but unfortunately you'll have to add the native unity support yourself.

I've integrated it for a client before, you'll have to use a nuget package manager for unity or import the dls individually.
